perl/README.md

```diff
@@ -0,0 +1,1021 @@
+# Bystro High Dimensional Genomic Variant and Sample Annotator Documentation
```

Comment:
This kind of sounds like it annotates samples instead of variants. Maybe it would be helpful to have a quick chat about this?